nautobot/ui/src/views/Login.jsx

```diff
@@ -12,16 +22,16 @@ export default function Login() {
         password: e.target.password.value,
       })
       .then(() => {
-        localStorage.setItem("nautobot-user", e.target.username.value)
-        window.location.replace("/")
+        refetchSession().then(() => {navigate("/")});
+
       })
       .catch(err => alert(err.detail))
   }
+
+  
   return (
-      <Card>
+      <Box boxShadow='base' p='6' rounded='md' bg='white'>
```

Comment:
Why is explicitly styling required here? Can this be done using a css import instead?

Reply:
This isn't CSS, it's control properties for the React component. But I worked with the design team to get some "standards" around these so I could provide a canonical/default box.